plsql/PlSqlParser.g4

```diff
@@ -526,7 +526,14 @@ drop_index
 
 create_table
     : CREATE (GLOBAL TEMPORARY)? TABLE tableview_name 
-        LEFT_PAREN column_name datatype (',' column_name datatype)*
+        LEFT_PAREN column_name datatype (NOT NULL)? (ENABLE | DISABLE)? (',' column_name datatype (NOT NULL)? (ENABLE | DISABLE)?)*
```

Comment:
Maybe extract duplicated syntax to separated rule? Something like this:

```ANTLR
column_name datatype_null_enable
   :  column_name datatype (NOT NULL)? (ENABLE | DISABLE)?
   ;
```

It improves readability, parse tree bypass, and clarity.